src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadataSettings.cs

```diff
@@ -45,6 +45,9 @@ public XbmcMetadataSettings()
         [FieldDefinition(6, Label = "Episode Images", Type = FieldType.Checkbox, Section = MetadataSectionType.Image, HelpText = "<filename>-thumb.jpg")]
         public bool EpisodeImages { get; set; }
 
+        [FieldDefinition(7, Label = "Disable Image Thumb Tags", Type = FieldType.Checkbox, Section = MetadataSectionType.Image, HelpText = "Do not include image thumb tags in <filename>.nfo (Requires 'Episode Images')", Advanced = true)]
+        public bool NoImageThumbTags { get; set; }
```

Comment:
A few things:

1. In keeping with the other options, this should be renamed to `EpisodeImageThumb` with the Label of `Episode Metadata Image Thumbs` and the default value set to true. This will be more inline with `SeriesMetadataEpisodeGuide` and `SeriesMetadataUrl`
2. Should to be moved to `4` (under `Episode Metadata`).
3. The reference to `Episode Images` should be changed to `Episode Metadata`

Reply:
Fixed